### PR TITLE
Update ralouphie/getallheaders dependency

### PR DESCRIPTION
**Pull request by jmleroux**
Thanks for this usefull test tool.
I had some dependency issues with `guzzlehttp/psr7` and found that we could update your package dependencies.

### CODE REVIEW

**Comment by jmleroux** [comment]
And it is tested: Travis use `2.0.5` with PHP 5.4 and 5.6, and `3.0.3` in PHP >= 5.6

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
 		"php": ">=5.4",
 		"ext-sockets": "*",
 		"ext-json": "*",
-		"ralouphie/getallheaders": "~2.0"
+		"ralouphie/getallheaders": "~2.0||^3.0.0"
```
Comment (donatj) [inline composer.json]:
```suggestion
		"ralouphie/getallheaders": "~2.0 || ~3.0"
```

For the sake of being picky let’s make them both of the same form.

Comment (jmleroux) [reply composer.json]: Done.
I will squash if validated

**Comment by donatj** [comment]
Thank you very much, I’ll tag a release very shortly.

**Comment by donatj** [comment]
It has been tagged! Thanks again.

https://github.com/donatj/mock-webserver/releases/tag/v2.1.0

**Pull request merged**